tutorials/ui/bbcode_in_richtextlabel.rst

```diff
@@ -812,13 +812,19 @@ Font options
 Named colors
 ~~~~~~~~~~~~
 
-For tags that allow specifying a color by name you can use names of the constants from
+For tags that allow specifying a color by name, you can use names of the constants from
 the built-in :ref:`class_Color` class. Named classes can be specified in a number of
 styles using different casings: ``DARK_RED``, ``DarkRed``, and ``darkred`` will give
 the same exact result.
 
 .. _doc_bbcode_in_richtextlabel_hex_colors:
```

Comment:
Should be moved down to above the "Hexadecimal color codes", or the title should be moved up